server/integration-tests/integration-tests-mysql/src/test/scala/com/prisma/integration/SettingUpChinookSpec.scala

```diff
@@ -65,19 +65,30 @@ class SettingUpChinookSpec extends FlatSpec with Matchers with IntegrationBaseSp
       importer.executeImport(json).await(50)
     }
 
-    importFile("nodes01.json")
-    importFile("nodes02.json")
-    importFile("relations01.json")
-    importFile("relations02.json")
-    importFile("lists01.json")
+//    importFile("nodes01.json")
+//    importFile("nodes02.json")
+//    importFile("relations01.json")
+//    importFile("relations02.json")
+//    importFile("lists01.json")
```

Comment:
Was this commenting intended?

Reply:
This is an ignored test that is only used to quickly have data for testing while developing.